vite.config.js

```diff
@@ -1,7 +1,16 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
+import dotenv from 'dotenv'
+
+dotenv.config();
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  server: {
+    port: Number(process.env.PORT)
+  },
+  define: {
+    'process.env.PORT': `${process.env.PORT}`
+  }
 })
```

Comment:
Maybe include the shortcut '@' for the file path.

Reply:
I have resolved this 🙇